app/panel/components/Panel.jsx

```diff
@@ -369,6 +384,30 @@ class Panel extends React.Component {
 				handleSignInClick={this._handlePromoSignInClick}
 				handleSubscribeClick={this._handlePromoSubscribeClick}
 				handleXClick={this._handlePromoXClick}
+				show
+			/>
+		);
+	}
+
+	/**
+	 * @returns {null|JSX}
+	 * @private
+	 * Renders the Insights promo modal if the user is not already an Insights subscriber
+	 */
+	_renderPlusPromoModal = () => {
+		if (this._plusSubscriber() || this._premiumSubscriber()) { return null; }
+
+		sendMessage('promoModals.sawPlusPromo', {});
+		sendMessage('ping', 'promo_modals_show_plus');
+
+		return (
+			<PlusPromoModal
+				handleGoAwayClick={this._handlePromoGoAwayClick}
+				handleSignInClick={this._handlePromoSignInClick}
+				handleSubscribeClick={this._handlePromoSubscribeClick}
```

Comment:
Won't `this._handlePromoSubscribeClick` open the checkout for Premium rather than plus?

Reply:
fixed!